### DISCUSSION TITLE
posts/master-react-native-5-open-source-projects-to-learn-from/

### DISCUSSION THREAD

**Discussion by giscus** [yjose/elazizi.com#104] [Blog posts]
# posts/master-react-native-5-open-source-projects-to-learn-from/

In this article, I will share with you the top 5 open-source React Native projects that you can learn from.

https://elazizi.com/posts/master-react-native-5-open-source-projects-to-learn-from/

**Comment by yassineDevox** [comment]
Thanks for sharing 🎉

**Comment by ahshaikh** [comment]
Thank you very much for sharing
